netty/src/test/java/io/grpc/netty/NettyServerHandlerTest.java

```diff
@@ -563,11 +562,11 @@ public void maxConnectionAge() throws Exception {
   @Test
   public void maxConnectionAgeGrace() throws Exception {
     maxConnectionAgeInNanos = TimeUnit.MILLISECONDS.toNanos(10L);
-    maxConnectionAgeGraceInNanos = TimeUnit.MILLISECONDS.toNanos(10L);
+    maxConnectionAgeGraceInNanos = TimeUnit.MILLISECONDS.toNanos(100L);
     setUp();
     createStream();
 
-    Thread.sleep(15L);
+    Thread.sleep(10L);
```

Comment:
It tests after the max age, during the grace period, the channel is still open. Without the sleep how can I get to the grace period?